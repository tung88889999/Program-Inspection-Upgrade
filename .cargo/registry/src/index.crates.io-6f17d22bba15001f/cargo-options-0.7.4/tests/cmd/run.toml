bin.name = "cargo-mimic"
args = "run --help"
