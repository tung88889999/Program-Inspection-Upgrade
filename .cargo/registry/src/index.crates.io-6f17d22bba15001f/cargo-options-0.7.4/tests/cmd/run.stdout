Run a binary or example of the local package

Usage: cargo-mimic[EXE] run [OPTIONS] [args]...

Arguments:
  [args]...  Arguments for the binary to run

Options:
  -q, --quiet                 Do not print cargo log messages
      --message-format <FMT>  Error format
  -v, --verbose...            Use verbose output (-vv very verbose/build.rs output)
      --color <WHEN>          Coloring: auto, always, never
      --config <KEY=VALUE>    Override a configuration value (unstable)
  -Z <FLAG>                   Unstable (nightly-only) flags to Cargo, see 'cargo -Z help' for
                              details
      --ignore-rust-version   Ignore `rust-version` specification in packages
  -h, --help                  Print help

Compilation Options:
  -j, --jobs <N>                Number of parallel jobs, defaults to # of CPUs
      --keep-going              Do not abort the build as soon as there is an error (unstable)
      --profile <PROFILE-NAME>  Build artifacts with the specified Cargo profile
      --target <TRIPLE>         Build for the target triple [env: CARGO_BUILD_TARGET=]
      --target-dir <DIRECTORY>  Directory for all generated artifacts
      --timings[=<FMTS>...]     Timing output formats (unstable) (comma separated): html, json
  -r, --release                 Build artifacts in release mode, with optimizations
      --unit-graph              Output build graph in JSON (unstable)

Feature Selection:
  -F, --features <FEATURES>  Space or comma separated list of features to activate
      --all-features         Activate all available features
      --no-default-features  Do not activate the `default` feature

Manifest Options:
      --frozen                Require Cargo.lock and cache are up to date
      --locked                Require Cargo.lock is up to date
      --offline               Run without accessing the network
      --manifest-path <PATH>  Path to Cargo.toml

Package Selection:
  -p, --package [<SPEC>]  Package to run (see `cargo help pkgid`)

Target Selection:
      --bin [<NAME>]      Run the specified binary
      --example [<NAME>]  Run the specified example

Run `cargo help run` for more detailed information.
